xbmc/interfaces/python/XBPython.cpp

```diff
@@ -581,6 +581,10 @@ bool XBPython::OnScriptInitialized(ILanguageInvoker *invoker)
       CLog::Log(LOGDEBUG, "PYTHONHOME -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
       CLog::Log(LOGDEBUG, "PYTHONPATH -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
     }
+
+#if defined(TARGET_DARWIN)
```

Comment:
Coming back to this one. SDLMain.mm is not so good because i wanted a spot where it works for all darwin platforms. I also selected this spot because the very same thing is already done for android in this file. You want me to find a "better" spot? In that case the android part will still be in this file ...